src/Pos/Generator/Block/Mode.hs

```diff
@@ -167,6 +180,18 @@ mkBlockGenContext bgcParams@BlockGenParams{..} = do
         bgcTxpMem <- (,ignoreTxpMetrics) <$> mkTxpLocalData
         bgcDelegation <- mkDelegationVar @SscGodTossing
         return BlockGenContext {..}
+  where
+    -- Genesis utxo is needed only for boot era stakeholders
+    bgcGenesisUtxo =
```

Comment:
this code looks like bulshit because of underlying db in `GStateContext` is initialized by other real `GenesisUtxo`, but you use some arbitrary `GenesisUtxo`.

Maybe I don't understand anything or you should take `GenesisUtxo` as argument of function